Cabal/doc/developing-packages.rst

```diff
@@ -33,38 +33,40 @@ them for us.
 Using "cabal init"
 ------------------
 
-The ``cabal init`` command is interactive. It asks us a number of
-questions starting with the package name and version.
+The ``cabal init --interactive`` command is interactive. If we answer
```

Comment:
Could you add a `TIP`[1] at the beginning of the "Quickstart" section above pointing to the (new) [Getting Started guide](https://cabal.readthedocs.io/en/latest/getting-started.html). Maybe something like:

```
.. TIP::
    If this is your first time using `cabal` you should check out the `Getting Started guide <https://cabal.readthedocs.io/en/latest/getting-started.html>`_.
```

That is probably the best "entrypoint" to the docs, but afaik many resources (and Search results still point the user here).

[1] https://docutils.sourceforge.io/docs/ref/rst/directives.html#id32

Reply:
Err, this should actually use a local reference (I don't know the exact syntax but you can find it in the reference I linked above).

Reply:
Thank you for your review. I can certainly get those two things accomplished today.

Reply:
> By "local reference" do you mean like a relative path/link, or something else?

Yeah, I think the following should work for linking to the other page:

```
`Getting Started guide <getting-started.html>`__
```

> or did you actually want a footnote?

Nah, inline link in the `.. TIP` is probably clearer.

Reply:
Okay, @m-renaud, I have done as you have instructed. Thank you for guiding me on this issue.

Reply:
LG, thanks!